components/shared/ui/menu.tsx

```diff
@@ -74,31 +120,8 @@ export const Menu: FC<Props> = props => {
             </button>
           </div>
           <div>
-            <ul className={`${smallMenuActive ? "flex" : "hidden"} md:flex flex-col font-medium md:flex-row h-full`}>
-              {props.item.elements.subitems.linkedItems.map((link, i) => (
-                <li key={i} className={`h-full px-5 ${mainColorBgClass[siteCodename]} group md:hover:bg-white`} onClick={() => handleMenuClick(i)}>
-                  {link.elements.subitems.value.length > 0 ? (
-                    <span>
-                      <DropdownButton item={link} />
-                      <div
-                        className={`${i === activeMenu ? "block" : "hidden"} md:group-hover:block absolute z-50 left-0 shadow-sm bg-white border-gray-200 w-full`}
-                      >
-                        <div className="grid max-w-screen-xl px-4 py-5 mx-auto text-gray-900 sm:grid-cols-2 md:grid-cols-3 md:px-6">
-                          <DropdownMenu links={link.elements.subitems.linkedItems} />
-                        </div>
-                      </div>
-                    </span>
-                  ) : (
-                    <Link
-                      {...link.elements.openInANewWindow.value[0] ? { rel: "noopener noreferrer", target: "_blank" } : {}}
-                      className="h-full flex items-center justify-between w-full py-2 pl-3 pr-4 font-medium text-gray-900 border-b border-gray-100 md:w-auto md:bg-transparent md:border-0"
-                      href={link.elements.externalLink.value ? link.elements.externalLink.value : "/" + link.elements.pageLink.linkedItems[0].elements.url.value}
-                    >
-                      {link.elements.label.value}
-                    </Link>
-                  )}
-                </li>
-              ))}
+            <ul className={`${smallMenuActive ? "flex" : "hidden"} flex-col md:flex md:gap-4 font-medium md:flex-row h-full`}>
```

Comment:
NITPICK: I kind of like it more without this md:gap-4 :)

Reply:
done